doc/dev/crimson/crimson.rst

```diff
@@ -220,37 +217,37 @@ Another SeaStore example::
     --seastore-secondary-devs /dev/sdb \
     --seastore-secondary-devs-type HDD
 
-You could stop the vstart cluster using::
+You may stop this ``vstart`` cluster by running::
```

Comment:
Let's remove modality entirely.

s/You may stop this ``vstart`` cluster by running::/Stop this ``vstart`` cluster by running/